frame/benchmarking/src/utils.rs

```diff
@@ -108,6 +112,51 @@ pub struct BenchmarkResults {
 	pub keys: Vec<(Vec<u8>, u32, u32, bool)>,
 }
 
+impl BenchmarkResult {
+	pub fn from_weight(w: Weight) -> Self {
+		Self { extrinsic_time: (w as u128) / 1_000, ..Default::default() }
+	}
+}
+
+/// Possible errors returned from the benchmarking pipeline.
+///
+/// * Stop: The benchmarking pipeline should stop and return the inner string.
+/// * WeightOverride: The benchmarking pipeline is allowed to fail here, and we should use the
+///   included weight instead.
+#[derive(Encode, Decode, Clone, PartialEq, Debug)]
+
+pub enum BenchmarkError {
+	Stop(#[codec(skip)] &'static str),
```

Comment:
Why do you skip this here?

Reply:
ah, at some point i was thinking about doing some storage stuff, but yeah not needed

Reply:
Use `sp_runtime::RuntimeString`

Reply:
Runtime string came with a whole host of other issues, again related to Arbitrary Error -> Static String, then Static String -> Runtime String, since it rust can't figure out to do two jumps.

Instead I removed `Encode, Decode` cause it just wasnt needed